gpii/configs/fm.ps.sr.dr.mm.os.lms.production.json

```diff
@@ -1,7 +1,7 @@
 {
```

Comment:
This config seems as production'y as we can currently get... perhaps it would be worth renaming this to something like:
trusted.flowmanager.production.json